nuget_witsml.md

```diff
@@ -0,0 +1,34 @@
+## Access pre-built Witsml nuget package
+
+If you only need to include the [Witsml](https://github.com/equinor/witsml-explorer/tree/main/Src/Witsml) project
+```sh
+witsml-explorer/Src/Witsml
+
+Witsml nuget package can be added to your own project using [nuget](https://docs.microsoft.com/en-us/nuget/). 
+It is necessary to author a new personal access token to be able to add the package to your project. A `nuget.config` (config file at solution, user or global scope) has to accommodate for username and the token afterwards.
+
+**Example** 
+
+1. Create your api access token at [https://github.com/settings/tokens](https://github.com/settings/tokens)
+2. Make those available through a config file at your preferred location (see: [configuring-nuget-behavior](https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior))
```

Comment:
those -> the token

(It's just a singular token, right?)

Reply:
You are right, I guess the github path ending in `/tokens` put me off :) Thanks!